src/core/p5.Graphics.js

```diff
@@ -60,6 +60,13 @@ p5.Graphics = class extends p5.Element {
 
     if (r === constants.WEBGL) {
       this._renderer = new p5.RendererGL(this.canvas, this, false);
+      const gl = this._renderer.GL;
+      const maxTextureSize = gl.getParameter(gl.MAX_TEXTURE_SIZE);
+      const maxAllowedPixelDimensions = Math.floor(
+        maxTextureSize / this._pixelDensity
```

Comment:
Ah yes, this depends on the density. I think this means that we may need to do a similar check when the user sets pixelDensity() too for complete coverage